### PR TITLE
Add more opengl tutorials

### PR DESCRIPTION
**Pull request by ShadowMitia**

### CODE REVIEW

File: resources/index.html [hunk 1]
```diff
@@ -46,6 +46,9 @@
 		* [Guide to Modern OpenGL Functions](https://github.com/fendevel/Guide-to-Modern-OpenGL-Functions)
 		* [Learn OpenGL](https://learnopengl.com/)
 		* [OpenGL 4.6 Specification](https://www.khronos.org/registry/OpenGL/specs/gl/glspec46.core.pdf)
+		* [OpenGL tutorials by Victor Gordan](https://www.youtube.com/playlist?list=PLPaoO-vpZnumdcb4tZc4x5Q-v7CkrQ6M-)
+		* [Modern OpenGL tutorials by Victor Gordan](https://www.youtube.com/playlist?list=PLPaoO-vpZnukvdD_iY8dCefCV6Jp0hJEv)
```
Comment (deccer) [inline resources/index.html]: twice the same link, one is enough

**Review by deccer — CHANGES_REQUESTED** [review]
One minor thing to complain about

File: resources/index.html (hunk 1 above)
Comment (ShadowMitia) [reply resources/index.html]: It's not the same link, it's two different playlist on the same channel. Maybe I could point to the channel directly?

Comment (deccer) [reply resources/index.html]: Leave out the 2nd, its just 3 videos nd other links provide more modern stuff already, maybe when victor adds more we can add it

**Review by deccer — APPROVED** [review]
LGTM

**Pull request merged**